components/bio-formats-plugins/src/loci/plugins/in/ImagePlusReader.java

```diff
@@ -570,59 +579,71 @@ private String constructSliceLabel(int ndx, IFormatReader r,
   {
     r.setSeries(series);
 
-    final int[] zct = r.getZCTCoords(ndx);
-    final StringBuffer sb = new StringBuffer();
-
-    int[] subC;
-    String[] subCTypes;
-    Modulo moduloC = r.getModuloC();
-    if (moduloC.length() > 1) {
-      subC = new int[] {r.getSizeC() / moduloC.length(), moduloC.length()};
-      subCTypes = new String[] {moduloC.parentType, moduloC.type};
-    } else {
-      subC = new int[] {r.getSizeC()};
-      subCTypes = new String[] {FormatTools.CHANNEL};
```

Comment:
Does it make sense to keep a way to include the modulo information in the slice labels?